conductor_api/src/dpki_instance.rs

```diff
@@ -0,0 +1,83 @@
+/// Defines a rust wrapper trait to match the Dpki exposed function trait definition
+/// for the conductor to use in the various dpki related flows, e.g. bootstrap and agent addition
+use crate::holochain::Holochain;
+use holochain_core_types::{
+    cas::content::{Address, AddressableContent},
+    error::HolochainError,
+};
+
+use holochain_core::nucleus::{
+    actions::call_zome_function::make_cap_request_for_call,
+    ribosome::capabilities::CapabilityRequest,
+};
+
+pub const DPKI_ZOME_NAME: &str = "dpki";
+pub const DPKI_TRAIT_FN_ADD_AGENT: &str = "create_agent_key";
+pub const DPKI_TRAIT_FN_INIT: &str = "init";
+pub const DPKI_TRAIT_FN_IS_INITIALIZED: &str = "is_initialized";
+
+use std::convert::TryInto;
+
+pub trait DpkiInstance {
+    fn dpki_cap_request(
```

Comment:
Not sure if this function actually needs to be in the trait?
It is only used internally to the implementations of the other functions right? Would rather have it be not part of the trait then since it is also not part of the zome trait.

Reply:
Done